admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,199 @@
+
+To run the tool on command line:
+
+```shell
+$ cluster-capacity --kubeconfig <path-to-kubeconfig> --podspec <path-to-pod-spec>
+```
+
+`--kubeconfig` is path to kubeconfig file to cluster and `--podspec` is path to sample pod spec
+that is used by the tool for estimation. The `podspec` specifies its resource requirements as
+`limits` or `requests`. The cluster capacity tool takes the pod's resource requirements into account
+for its estimation analysis. An example of input pod spec (`pod.yaml`) is given below:
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+```
+
+
+The output of the cluster capacity command is a number of estimated pods that can be scheduled on the cluster.
+`--verbose` flag can also be provided as part of the above command to output detailed description
+about how many pods can be scheduled on each node in the cluster. An example of verbose output on
+a 2-node cluster is as follows:
+
+```shell
+small-pod pod requirements:
+	- CPU: 150m
+	- Memory: 100Mi
+
+The cluster can schedule 52 instance(s) of the pod small-pod.
+
+Termination reason: Unschedulable: No nodes are available that match all of the following predicates:: Insufficient cpu (2).
+
+Pod distribution among nodes:
+small-pod
+	- 192.168.124.214: 26 instance(s)
+	- 192.168.124.120: 26 instance(s)
+```
+
+[admin-guide-cluster-capacity-job]
+### Running cluster capacity as a Job
+
+Below is an example of a Job spec (`cluster-capacity-job.yaml`) that runs cluster capacity image as a job in a pod.
+Note that input pod spec (`pod.yaml`) to cluster capacity analysis is mounted in a volume using a `ConfigMap`.
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cluster-capacity-job
+spec:
+  parallelism: 1
+  completions: 1
+  template:
+    metadata:
+      name: cluster-capacity-pod
+    spec:
+        containers:
+        - name: cluster-capacity
+          image: openshift/origin-cluster-capacity
+          imagePullPolicy: "Never"
+          volumeMounts:
+          - mountPath: /test-pod
+            name: test-volume
+          env:
+          - name: CC_INCLUSTER
+            value: "true"
+          command:
+          - "/bin/sh"
+          - "-ec"
+          - |
+            /bin/cluster-capacity --podspec=/test-pod/pod.yaml --verbose
+        restartPolicy: "Never"
+        serviceAccountName: cluster-capacity-sa
+        volumes:
+        - name: test-volume
+          configMap:
+            name: cluster-capacity-configmap
+```
+
+Also note that the above pod spec specifies the following environment variable which is required to
+let the cluster capacity command know that it is running inside a cluster as a pod.
+
+```yaml
+env:
```

Comment:
@derekwaynecarr Is this intended for @aveshagarwal ? I'm not sure what you mean.